components/search/MoreFiltersDropdown.vue

```diff
@@ -119,6 +150,17 @@
         return Object.assign({}, this.selected);
       },
 
+      dateFilterSelected(facetName, selectedFields) {
+        const getRange = rangeToQueryParam(selectedFields);
+        let newArray = [];
+
+        if (getRange !== '[* TO *]') {
```

Comment:
This function shouldn't need to know about the structure of the range query.

Replace it with:
```js
      dateFilterSelected(facetName, dateRange) {
        const rangeQuery = (!dateRange.start && !dateRange.end) ? [] : [rangeToQueryParam(dateRange)];

        this.updateSelected(facetName, rangeQuery);
      },
```